src/main/scala/org/musigma/sbt/rat/SbtRatPlugin.scala

```diff
@@ -37,93 +39,163 @@ object SbtRatPlugin extends AutoPlugin {
   override def requires = JvmPlugin
 
   object autoImport {
-    // FIXME: this should probably be a task, not a configuration (e.g., packageBin)
-    val Audit = config("audit")
+    val ratCheck = taskKey[Unit]("Performs a release audit check")
 
-    val auditCheck = taskKey[Unit]("Performs a release audit check")
-    val auditReport = taskKey[AuditReport]("Generates a release audit report")
+    type AuditReport = ClaimStatistic
```

Comment:
If you're renaming the tasks, you should rename this type alias to `RatReport` as well.

Reply:
Push a fix for this change